docs/configuration/model.md

```diff
@@ -4,7 +4,7 @@
 
```

Comment:
We should revert those changes. (except grammar fixes of courses)

Reply:
Yep, agreed